core/src/main/java/org/lflang/generator/python/PythonGenerator.java

```diff
@@ -578,7 +573,7 @@ private static String setUpMainTarget(
             add_subdirectory(core)
             set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR})
             set(LF_MAIN_TARGET <pyModuleName>)
-            find_package(Python 3.7.0...<3.11.0 COMPONENTS Interpreter Development)
+            find_package(Python 3.10.0...<3.11.0 REQUIRED COMPONENTS Interpreter Development)
```

Comment:
Should this be <3.12.0?

Reply:
No, this does specifically not add Python 3.11 support. For this we should finalize https://github.com/lf-lang/lingua-franca/pull/1902.

Reply:
I tried what happens if we just enable Python 3.11 on top of this branch in https://github.com/lf-lang/lingua-franca/pull/1915, but we are getting segfaults. So we still need the fixes in https://github.com/lf-lang/lingua-franca/pull/1902.